### ISSUE TITLE
 Incorrect flamethrower stats

### ISSUE THREAD

**Issue by ETLegacyTeam** [etlegacy/etlegacy#453]
Author: @rmarquis (Spyhawk)
Date: 2013-12-30
Redmine Issue: 453, https://dev.etlegacy.com/issues/453

---

The Flamethrower stats are bugged, due to an incorrect count of fired "ammunition".

See the screenshot in attachment: Killed 5 bots in one go (+myself), 110 hits but only 1 fired shot counted. Accuracy: 11000%.
This bug is fixed in ETPub/NQ.

---

Related commit(s):
c15cf6e57dc1f56cbd15b21c161a07bb6e94b00d - cgame: cap stats at 100%, refs 453
 b35a686da4faed9f1702f6d934add71a8f9edcb7 - game: fix flamer deaths getting blamed on the worl..

**Comment by ETLegacyTeam** [comment]
From: @IR4T4 (IR4T4)
Date: 2013-12-30T05:22:59Z

---

Beside fixing the above bug we should cap accuracy (for views only).

**Comment by ETLegacyTeam** [comment]
From: @IR4T4 (IR4T4)
Date: 2014-01-24T17:47:44Z

---

 Spyhawk, can you tell us how to reproduce this?

**Comment by ETLegacyTeam** [comment]
From: @rmarquis (Spyhawk)
Date: 2014-01-24T18:05:21Z

---

Keep in mind that stats are now capped, so you won't see stats above 100% anymore, but the fired shots counter will still be inconsistent.

Furthermore, the screenshot included above seems to depict a somewhat extreme case. I did later test later on, and realized that the fired shots counter isn't always wrong - sometimes it is fully correct, most of the time is it almost accurate, and in some cases it is plain wrong. I'm now also unsure if this has been fixed in NQ/ETPub or not.

The case depicted in the screenshot was a single, long flamethrower 'shot' (probably more than half a tank, or 100+ flame 'bullets') that did 5 kills in a row, and also killed myself.
